.travis.yml

```diff
@@ -13,10 +13,12 @@ notifications:
   webhooks:
     - http://savage.nonblocking.io:8080/savage/travis
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
It might be worth adding a version check for yarn in one of the other files, and expose a warning for when a new yarn version is released but xenial's built-in version stays behind. Late PR maybe?

Reply:
This isn't a common problem, since we're backward compatible with older versions up to `1.10.1`, which made a breaking change. (The Trusty image had a really old version). The current minimum version is listed at:

https://github.com/ampproject/amphtml/blob/49baa09370fe62b32359dda5826244804288d928/package.json#L9

It might be possible to reuse this code to print a warning on Travis. Low priority IMO, given the above check.

https://github.com/ampproject/amphtml/blob/48e77a106b451ec0cc64bff689fc805031a60af8/build-system/check-package-manager.js#L128-L161